sros2/sros2/verb/generate_artifacts.py

```diff
@@ -44,8 +48,43 @@ def add_arguments(self, parser, cli_name):
 
     def main(self, *, args):
         try:
-            success = generate_artifacts(
+            success = _generate_artifacts(
                 args.keystore_root_path, args.security_contexts, args.policy_files)
         except FileNotFoundError as e:
             raise RuntimeError(str(e))
         return 0 if success else 1
+
+
+def _generate_artifacts(keystore_path=None, identity_names=[], policy_files=[]):
```

Comment:
Note that using lists as default values should probably be fixed in a follow-up.